ChangeLog.md

```diff
@@ -1,9 +1,10 @@
 ## Client Versions ##
-* v9.51.5 (2015-10-22)
+* v9.52.0 (2015-10-28)
```

Comment:
Initially that's what it was, but this functionality requires a reparse to populate `messages_hash`, so I incremented the minor version.